src/SourceBuild/patches/sdk/0001-Disable-analyzers-for-dotnet-watch.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Wed, 8 Mar 2023 14:51:59 -0600
+Subject: [PATCH] Disable analyzers for dotnet-watch.
+
+---
```

Comment:
Please add a `Backport` link.  If not already done, please backport this right away.

Reply:
Backport is https://github.com/dotnet/sdk/pull/31076, opened https://github.com/dotnet/installer/pull/15743 to add that to the patch.